openapi/openapi_utils_test.go

```diff
@@ -554,6 +554,19 @@ func TestFindPermission(t *testing.T) {
 			RequestedPath: "/without/trailing/slash/",
 			Method:        "POST",
 		}, matchedPath)
+
+		found, matchedPath, err = oas.FindPermission(OASRouter, "/ignore/trailing/slash/", "GET")
```

Comment:
agree, I didn't do it because I was worried about the file being used somewhere else with assertions on exposed paths, I just checked and apparently not so I'm adding it

Reply:
i'm keeping the other endpoint changed anyhow as an additional test case